### PR TITLE
feat: implement wallet aggregator sdk for Injective chains

### PR DESCRIPTION
**Pull request by M-Picco**
Implement wallet aggregator sdk for Injective chain.

This PR is a fragment split off https://github.com/XLabs/portal-bridge-ui/pull/47 and updated with latest changes.

### CODE REVIEW

File: src/contexts/InjectiveWalletContext.tsx [hunk 1]
```diff
@@ -1,127 +1,53 @@
...
+import {
+  InjectiveWallet,
+  KeplrWallet,
+} from "@xlabs-libs/wallet-aggregator-injective";
+import { useWallet } from "@xlabs-libs/wallet-aggregator-react";
+import { CHAIN_ID_INJECTIVE } from "@xlabs-libs/wallet-aggregator-core";
+import { getNetworkInfo } from "@injectivelabs/networks";
+
+export const getInjectiveWallets = () => {
+  if (!["mainnet", "testnet"].includes(process.env.REACT_APP_CLUSTER || ""))
```
Comment (raop155) [inline src/contexts/InjectiveWalletContext.tsx:15]: This validation can be extract to a `variable`

**Review by raop155 — APPROVED** [review]
LGTM

File: src/contexts/InjectiveWalletContext.tsx (hunk 1 above)
Comment (M-Picco) [reply src/contexts/InjectiveWalletContext.tsx:15]: Do you mean extract the `["mainnet", "testnet"]` to a constant in the outer scope?

Comment (M-Picco) [reply src/contexts/InjectiveWalletContext.tsx:15]: Ah! I see now. I think it's clearer that way, thanks for pointing it out.

**Pull request merged**